src/styles/card.scss

```
@@ -0,0 +1,102 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  color: $main-color;
+  font-style: normal;
+  font-size: 10px;
+  font-weight: 400;
```

Default value, so you can just remove it everywhere
```suggestion
  font-size: 10px;
```